packages/evm/src/opcodes/functions.ts

```diff
@@ -171,12 +181,25 @@ export const handlers: Map<number, OpHandler> = new Map([
     0x0a,
     function (runState) {
       const [base, exponent] = runState.stack.popN(2)
-      if (exponent === BigInt(0)) {
-        runState.stack.push(BigInt(1))
+      if (base === BIGINT_2) {
```

Comment:
Is this conditional added to pre-compute some commonly used exponents of base 2?